src/commands/open/open.js

```diff
@@ -0,0 +1,38 @@
+const { createOpenAdminCommand, openAdmin } = require('./open-admin')
+const { createOpenSiteCommand, openSite } = require('./open-site')
+
+/**
+ * The open command
+ * @param {import('commander').OptionValues} options
+ * @param {import('../base-command').BaseCommand} command
+ */
+const open = async (options, command) => {
+  if (!options.site || !options.admin) {
+    command.help()
+  }
+
+  if (options.site) {
+    await openSite(options, command)
+  }
+  // Default open netlify admin
+  await openAdmin(options, command)
```

Comment:
Not sure why this is not working, it seems `commander` exits after the `help` method

Reply:
Yea `command.help()` shows the help and exits I refactored to `log(command.helpInformation())` this returns the help information as string.